apps/next/pages/_app.tsx

```diff
@@ -14,6 +15,12 @@ function MyApp({ Component, pageProps }: SolitoAppProps) {
           content="Expo + Next.js with Solito. By Fernando Rojo."
         />
         <link rel="icon" href="/favicon.ico" />
+        <link
```

Comment:
Wouldn't it be better to use expo font loading? I think it works both on web and mobile